Gemfile.lock

```diff
@@ -242,96 +242,95 @@ GEM
       rest-client (>= 1.6.7)
     marcel (0.3.3)
       mimemagic (~> 0.3.2)
-    method_source (0.9.2)
-    mime-types (3.3)
+    method_source (1.0.0)
+    mime-types (3.3.1)
```

Comment:
### [mime-types](https://github.com/mime-types/ruby-mime-types/)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 3.3 → 3.3.1

[_[change-log](https://github.com/mime-types/ruby-mime-types/blob/master/History.md), [source-code](https://github.com/mime-types/ruby-mime-types/)_]





<details>
<summary>Commits</summary>

A change of **5** commits. See the full changes on [the compare page](https://github.com/mime-types/ruby-mime-types/compare/v3.3...v3.3.1).

These are the individual commits:
- (58c658d) [CI: Drop unused sudo: false Travis directive](https://github.com/mime-types/ruby-mime-types/commit/58c658ddee8770c8bd8cb5fd3e92ece8230806a7)
- (1787454) [Merge pull request #142 from olleolleolle:patch-1](https://github.com/mime-types/ruby-mime-types/commit/178745421b7a0664bf7ff33885c3fb38b6597fb7)
- (c446731) [Fix [_1,_2,_3] parameters (conflict with Ruby 2.7.0); Update…](https://github.com/mime-types/ruby-mime-types/commit/c44673179d24e495e5fb93282a87d37f09925d25)
- (a9b7dd4) [Prepare for 3.3.1 release](https://github.com/mime-types/ruby-mime-types/commit/a9b7dd4877de3863ecb63b8c7e8122e128228e73)
- (2969a1a) [Update gemspec for 3.3.1 release](https://github.com/mime-types/ruby-mime-types/commit/2969a1a788a2d891e2bd16d0027f84084835297e)


</details>